api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/standalone/StandaloneLoaderService.java

```diff
@@ -61,30 +61,42 @@ public void initializeCache() {
 
     private void loadApplicationCache() {
         File[] appFiles = getFiles(servicesDirectory + "/apps");
+        log.debug("Found {} files", appFiles.length);
         if (appFiles.length == 0) {
             log.error("No service definition files found.");
             return;
         }
 
         for (File file : appFiles) {
-            createContainerFromFile(file);
+            log.debug("Processing file {}", file.getName());
+            try {
+                createContainerFromFile(file);
+            } catch (Exception e) {
+                log.error("Failed to load file {} because {}", file.getName(), e.getMessage());
+            }
         }
     }
 
     private void loadOpenAPICache() {
         File[] openAPIFiles = getFiles(servicesDirectory + "/apiDocs");
+        log.debug("Found {} API Doc files", openAPIFiles.length);
         if (openAPIFiles.length == 0) {
             log.error("No apiDocs files found.");
             return;
         }
 
         for (File openAPIFile : openAPIFiles) {
-            loadApiDocCache(openAPIFile);
+            log.debug("Processing {}", openAPIFile.getName());
+            try {
+                loadApiDocCache(openAPIFile);
+            } catch (Exception e) {
+                log.error("Fail to load API Doc from {} because {}", openAPIFile.getName(), e.getMessage());
```

Comment:
maybe we should use the apimlLog and the apiml messages.
i.e. `org.zowe.apiml.apicatalog.apidocRetrievalProblem`
I know it's a standalone component but that would provide more information